example/example.documentation.yaml

```diff
@@ -547,6 +547,71 @@ paths:
                     status: error
                     error:
                       message: Sample error message
+  /v1/serialize:
+    get:
+      operationId: GetV1Serialize
+      responses:
+        "200":
+          description: GET /v1/serialize Positive response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    const: success
+                  data:
+                    type: object
+                    properties:
+                      map:
+                        type: array
+                        items:
+                          type: array
+                          prefixItems:
+                            - type: string
+                            - type: boolean
+                          items:
+                            not: {}
+                      set:
+                        type: array
+                        items:
+                          type: number
+                          format: double
+                          minimum: -1.7976931348623157e+308
+                          maximum: 1.7976931348623157e+308
```

Comment:
There seems to be an issue for YAML with large exponential numbers.
Should be investigated. Didn't notice/happen before

```
example/example.documentation.yaml
┌─────────┬──────────┬──────────┬───────┬───────────────────────────────────┐
│ (index) │   line   │ severity │ code  │              message              │
├─────────┼──────────┼──────────┼───────┼───────────────────────────────────┤
│    0    │ '581:35' │ 'error'  │ 10014 │ "maximum' value must be a number" │
│    1    │ '580:35' │ 'error'  │ 10015 │ "minimum' value must be a number" │
└─────────┴──────────┴──────────┴───────┴───────────────────────────────────┘
```